tests/inc/RequestTest.php

```diff
@@ -425,7 +425,7 @@ public function schemeProvider()
     public function ifIdentificationDoesNotHaveTwoPartsExceptionIsThrown()
     {
         $request = new \Request($this->config, ['HTTPS' => 'on']);
-        $request->identifyUser(null, 'This is a bad header');
+        $request->identifyUser('This is a bad header');
```

Comment:
See the comment on the ```Request```-class.

Reply:
Which comment would that be? As part of this PR, the params were switched for `identifyUser` so that the param with a default is last in the list.